finder-app/makefile

```diff
@@ -0,0 +1,35 @@
+INCLUDE_DIRS = 
+LIB_DIRS = 
+CROSS_COMPILE=gcc
+CC=gcc
```

Comment:
@rajatchaple use 
CC=$(CROSS_COMPILE)gcc and then replace CROSS_COMPILE below with CC.  That way when CROSS_COMPILE is not defined you will just use native compile.  As you've defined it here you need CROSS_COMPILE to be the full executable, not just the prefix.  This will be important to fix for the next assignment when we are actually cross compiling in the automated test.

Reply:
Thank you Dan for the inputs.
I have made the changes as directed above.